netbox_netprod_importer/vendors/cisco/nxos.py

```diff
@@ -83,13 +82,11 @@ def _guess_type_from_if_type(self, interface):
         raise TypeCouldNotBeParsedError()
 
     def _get_ifstatus_by_if(self):
-        cmd = "show interface status | json"
+        cmd = "show interface status" + self._driver_end_selection()
 
         if not self.cache.get("ifstatus"):
             status_conf_dump = self.device.cli([cmd])[cmd]
-            status = json.loads(
-                status_conf_dump
-            )["TABLE_interface"]["ROW_interface"]
+            status = self._correct_and_convert_to_dict(status_conf_dump)["TABLE_interface"]["ROW_interface"]
```

Comment:
Same than before about the 79char/l

Reply:
Of course I can, I just use the PEP8 standards.
If for you this is fundamentally changed.